swim/handlers.go

```diff
@@ -21,15 +21,17 @@
 package swim
 
 import (
+	"errors"
+
 	log "github.com/uber-common/bark"
-	"github.com/uber/ringpop-go/swim/util"
+	"github.com/uber/ringpop-go/util"
 	"github.com/uber/tchannel-go/json"
 	"golang.org/x/net/context"
 )
 
-// An Arg is a blank argument used as filler for making TChannel calls that require
-// nothing to be passed to Arg3
-type Arg struct{}
+// An arg is a blank argument used as filler for making TChannel calls that require
+// nothing to be passed to arg3
+type arg struct{}
```

Comment:
I came across this name today and had to read some code to understand what it is for.
Do you think that emptyArg3 is a better name?